src/app/results/results.component.css

```diff
@@ -365,41 +365,141 @@ div.autocorrect{
   margin-left: 6%;
 }
 
-@media screen and (min-width:1920px) {
-  #tools {
-    margin-left: 27%;
+
+@media screen and (max-width: 1920px) {
+  #search-options {
+    margin-left: 5.5%
+  }
+  div.result{
+    margin-left: -5.7%;
+  }
+  div.autocorrect{
+    margin-left: 0;
+  }
+  div.result.message-bar{
+    margin-left: 0;
+  }
+  div.feed.col-md-10.col-md-offset-1.col-sm-10.col-sm-offset-1.col-lg-10.col-lg-offset-1 {
+    width: 34%;
   }
 }
 
-@media screen and (max-width: 1440px) {
-  .feed {
-    width: 43%;
+@media screen and (max-width: 1800px) {
+  #search-options {
+    margin-left: 6.5%
+  }
+  div.result{
+    margin-left: -2.7%;
+  }
+  div.autocorrect{
+    margin-left: 1%;
+  }
+  div.result.message-bar{
+    margin-left: 1%;
+  }
+  div.feed.col-md-10.col-md-offset-1.col-sm-10.col-sm-offset-1.col-lg-10.col-lg-offset-1 {
+    width: 36%;
   }
 }
+@media screen and (max-width: 1750px){
```

Comment:
@nikhilrayaprolu What you said is partially correct, but there are no finite break points since we use percentages, the borders constantly vary, however I have found a solution, I will not use standard bootstrap classes. Please refer new pull request